api/app/controllers/spree/api/variants_controller.rb

```diff
@@ -70,6 +70,12 @@ def scope
         def variant_params
           params.require(:variant).permit(permitted_variant_attributes)
         end
+
+        def include_list
+          [{ option_values: :option_type }, :product, :default_price,
+           Spree::Config.variant_gallery_class.preload_params,
+           { stock_items: :stock_location }]
```

Comment:
The readability of this could be improved by putting one thing per line:

```Ruby
[
  :default_price,
  :product,
  { option_values: :option_type },
  { stock_items: :stock_location },
  Spree::Config.variant_gallery_class.preload_params
]
```